.circleci/config.yml

```diff
@@ -101,7 +111,7 @@ jobs:
       - attach_workspace:
           at: ~/player
       # need to filter because //android/demo rules dont resolve
-      - run: bazel build --config=ci -- $(bazel query 'filter("native_bundle$", //...)')
+      - run: bazel build --config=ci --define build_type=$BUILD_TYPE -- $(bazel query 'filter("native_bundle$", //...)')
```

Comment:
I don't think you need the `build_type` here since we aren't compiling anything that uses this